main.py

```diff
@@ -57,14 +57,17 @@ def _run(self):
 
             # Yeey this post probably isn't breaking the rules 🌈
             try:
-                if self.config.SEND_ARTIST_REPLY is True and exceeds_threshold is False and parsed_submission.success is True:
-                    self._reply_with_sticky_post(submission, self.last_fm.get_artist_reply(parsed_submission))
-            except (pylast.WSError, EmptyDescription):
-                # Can't find artist or description is empty, logging
-                self.database.log_error_causing_submission(parsed_submission, submission, traceback.format_exc())
+                if exceeds_threshold is False and parsed_submission.success is True:
+                    self._reply_with_sticky_post(submission, DescriptionProvider().get_reply(parsed_submission))
```

Comment:
Any reason to not reuse the DescriptionProvider instance rather than recreating it?  Doesn't really hurt us on the performance side as we don't need it constantly anyway more of a curiosity thing.

Reply:
You're right, now it initializes every time which is unnecessary